test/wdio/tsconfig-stencil.json

```diff
@@ -35,6 +35,8 @@
     "./global.ts"
   ],
   "exclude": [
-    "./**/*.test.tsx"
+    "./**/*.test.tsx",
+    "./global-script/**/*.tsx",
+    "./global-script/**/*.ts"
```

Comment:
This is just to ensure that `global-script` stuff isn't included in the existing project

Reply:
Can we turn this into a comment that lives above this configuration? That way folks who modify this file next can get a little additional context when making changes

Reply:
sure, makes sense